docs/types_of_spectra.rst

```diff
@@ -42,7 +44,11 @@ with their corresponding ``specutils`` representations:
    not rectangular), this case does not have a specific representation in
    ``specutils``.  Instead, this case should be dealt with by making lists (or
    numpy object-arrays) of `~specutils.Spectrum1D` objects, and iterating over
-   them.
+   them. Specutils does provide a `SpectrumList` class which is
+   primarily intended for enabling data loaders that need to read multiple
+   heterogenous spectra (see :ref:`multiple_spectra`). Users should rarely need
```

Comment:
I think you should add something here to indicate in what way it's different from a `list` (i.e., almost not at all).  I think you might be able to just get away with something like "Specutils does provide a SpectrumList class, but it is gives no user functionality beyond a list.  Rather, it enables data loaders..."

That is, right now when I read this trying to pretend to be a poor user, I was more confused because it leads me to think "well, if it's useful to them for loading, maybe it'll be useful to me - it must have more stuff after all!"